src/api/theMovieDB.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Fetch movie's data from themoviedb api
+ *
+ * @param {number} page - Number of the page to query
+ * @returns {object}
+ */
+async function getMovies(page) {
+  const respnse = await fetch(
+    `https://api.themoviedb.org/3/discover/movie?api_key=${process.env.MOVIE_API_KEY}&page=${page}`
```

Comment:
you should grab the API key in a safer way, imagine the scenario where you forget to add it, the base url should be in a constant

Reply:
Separate the base `url` and `parcel` already use `dotenv`
https://parceljs.org/env.html

Do you mean that I need to handle an error if the `.env` file doesn't exist? 😅